packages/web-components/package.json

```diff
@@ -42,14 +42,14 @@
   "devDependencies": {
     "@microsoft/api-extractor": "^7.47.9",
     "@open-wc/testing": "^4.0.0",
-    "@rollup/plugin-commonjs": "^28.0.0",
```

Comment:
Could not built locally so bumped all stuff needed to built on a newer node version